src/westpa/cli/tools/w_timings.py

```diff
@@ -0,0 +1,112 @@
+from datetime import timedelta
+
+import numpy as np
+
+from westpa.tools import (
+    WESTTool,
+    WESTDataReader,
+    IterRangeSelection,
+)
+
+
+# NumPy fixed time units (excludes nonlinear units 'Y and 'M')
+TIME_UNITS = ('W', 'D', 'h', 'm', 's', 'ms', 'us', 'ns', 'ps', 'fs', 'as')
+
+
+def _unit(delta):
+    # Return the unit of a NumPy timedelta.
+    words = str(delta.dtype).split('[')
+    if len(words) == 1:
+        return None
+    return words[1][:-1]
+
+
+def _str(delta):
+    # Return a string representation of a NumPy timedelta.
+    # Example: timedelta64(5380000, 'ps') -> '5.38 us'
+    if _unit(delta) is None:
+        return str(delta)
+    for unit in TIME_UNITS:
+        unit_delta = np.timedelta64(1, unit)
+        try:
+            if delta >= unit_delta:
+                break
+        except OverflowError:
+            continue
+    return f'{delta / unit_delta} {unit}'
+
+
+def _delta(arg):
+    # Construct a NumPy timedelta from an argument string.
+    # Example: '100_ps' -> timedelta64(100, 'ps')
+    try:
+        value, unit = arg.split('_')
+    except ValueError:
+        raise ValueError('must be formatted as <value>_<unit>')
+    if unit not in TIME_UNITS:
+        raise ValueError(f'{unit!r} is not a recognized time unit')
+    return np.timedelta64(int(value), unit)
+
+
+class WTimings(WESTTool):
+    prog = 'w_timings'
+    description = 'Print timing information for a WESTPA simulation.'
+
+    def __init__(self):
+        super().__init__()
+        self.data_reader = WESTDataReader()
+        self.iter_range = IterRangeSelection(self.data_reader)
+        self.tau = None
+
+    def go(self):
+        start = self.iter_range.iter_start - 1
+        stop = self.iter_range.iter_stop - 1
+        with self.data_reader:
+            iter_summaries = self.data_reader.we_h5file['summary'][start:stop]
+
+        walltime = iter_summaries['walltime'].sum()
+        cputime = iter_summaries['cputime'].sum()
+        n_particles = iter_summaries['n_particles'].sum()
+        n_iters = len(iter_summaries)
+
+        print('Iterations:', n_iters)
```

Comment:
Probably would be easier to read if we pad the labels so the values are all aligned like with pstatus in sim_manager.

https://github.com/westpa/westpa/blob/westpa2/src/westpa/core/sim_manager.py#L205-L210